src/current/v23.2/known-limitations.md

```diff
@@ -10,100 +10,61 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/92961)
+### SQL Optimizer and Read Committed Isolation
 
-### Low estimated Request Units are rounded to zero
+#### Optimizer and Locking Behavior
 
-The [Request Units](https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster-serverless#request-units) (RUs) estimate surfaced in [`EXPLAIN ANALYZE`]({% link {{ page.version.version }}/explain-analyze.md %}) is displayed as an integer value. Because of this, fractional RU estimates, which represent very inexpensive queries, are rounded down to zero.
+- The SQL optimizer has limitations under certain isolation levels:
+  - The new implementation of `SELECT FOR UPDATE` under serializable isolation is not yet the default setting. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114737)
+  - `SELECT FOR UPDATE` does not lock completely-NULL column families in multi-column-family tables. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/116836)
+  - `SELECT ... FOR UPDATE` and `SELECT ... FOR SHARE` locks are dropped on lease transfers and range splits/merges. Unreplicated locks, used by default under SERIALIZABLE isolation, can be dropped during such events, affecting the desired ordering of concurrent accesses and potentially leading to transaction retry errors. [More information on SELECT FOR UPDATE](https://www.cockroachlabs.com/docs/v23.2/select-for-update)
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/100617)
+#### Read Committed Isolation Limitations
 
-### Statistics for deleted tables in `system.table_statistics` do not get removed
+- Several capabilities are not yet supported with Read Committed isolation:
+  - DDL statements cannot be performed within explicit Read Committed transactions. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114778)
+  - Specific unique checks are not supported under Read Committed isolation. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110873)
+  - `SKIP LOCKED` requests do not reliably skip over replicated locks. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110743)
+  - Promoting a lock from `shared` to `exclusive` is not supported. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110435)
+  - Schema changes are not allowed within explicit transactions, and certain unique constraints are not supported. Shared locks cannot be promoted to exclusive locks, and `SKIP LOCKED` requests do not reliably skip over replicated locks. [More information on READ COMMITTED](https://www.cockroachlabs.com/docs/v23.2/read-committed)
+  - Multi-column-family checks during updates may not function correctly. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/112488)
```

Comment:
This wording is vague.

```suggestion
  - Multi-column-family checks during updates may not function correctly. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/112488)
```

Reply:
Thanks. Posted a question [here](https://github.com/cockroachdb/cockroach/issues/112488#issuecomment-1889680069).

Reply:
Instead of "may not function correctly" I suggest "are not supported under read committed isolation". I [answered your question](https://github.com/cockroachdb/cockroach/issues/114605#issuecomment-1890059871), hopefully that helps.